_pages/welcome-to-TTS/getting-started.md

```diff
@@ -0,0 +1,255 @@
+You can also look through the Handbook issues and fix something.
+
+Remember: you can be fearless. If you ever manage to break something in Github it can easily be changed back to how it was --- that's the beauty of version control. 
+If you get stuck, you can just wave in the #[tts-handbook](https://gsa-tts.slack.com/app_redirect?channel=tts-handbook) Slack channel and someone will quickly help you.
+
+### Code Reviews
+
+We believe you have the power to create the changes you want to see. 
+Those changes often result in Github pull requests, which need a code review.
+
+Whether you are an engineer or not, sooner or later you can expect to probably take part in a code review, 
+whether that is because of a project you're working on, or maybe your subject matter expertise is relevant.
+
+If pull requests and code reviews feel a bit foreign to you, it
+might be helpful to think of a pull request as sharing a draft Google Doc, 
+and a code review is something like suggesting changes in a shared Doc.  
+In both cases, you're collaboratively trying to reach a final version.
+
+Code reviews don't have to be scary: 
+[the 18F development guide has some great guidance](https://github.com/18F/development-guide/tree/master/code_review#tips-for-a-successful-review) 
+on how to be a good code review partner. 
+The main thing to remember is simply to be kind. We're all trying to improve things.
+
+
+### Setting up your Mac for success
+
+For most people at TTS, sooner or later you'll probably find yourself wanting to work with a 
+project locally on your Mac. 
+We recommend using our [laptop script](https://github.com/18F/laptop#what-it-sets-up) 
+as a quick way to set your Mac up and use our technology stack properly.
+
+If you're not an engineer, it's possible you may not need it, but when you do you'll be glad to know where it is. 
+Just follow the steps: everything will be set up correctly and you'll minimize weird conflicts and problems later.
+
+If you are an engineer, we understand that your preferences about optimal development environments may differ from the laptop script. 
+In that case, please avoid setting up your own environment until after we've talked about it in the [#dev](https://gsa-tts.slack.com/messages/C02CD5VUQ) channel. 
+Maybe you've identified a good way to update the script!
+
+If you are running a project locally, you may also find yourself needing to know 
+[some basics of the Mac OS X command line](https://blog.teamtreehouse.com/introduction-to-the-mac-os-x-command-line).
+
+And remember, if you can't figure out how to get it running, that's probably not your fault --- poor documentation is a defect. 
+You can file an issue in the repository or contact one of the project owners or contributors and ask them to update their README and any other relevant documentation.
+
+
+### Markdown
+
+[Markdown](https://www.markdownguide.org/cheat-sheet/) is probably the most commonly used way to 
```

Comment:
Minor nit - should be a dash in commonly-used

Reply:
It's an adverb, right? I don't think that's right https://www.dailywritingtips.com/adverbs-and-hyphens/